x-pack/plugins/osquery/public/fleet_integration/osquery_managed_policy_create_import_extension.tsx

```diff
@@ -148,14 +148,13 @@ export const OsqueryManagedPolicyCreateImportExtension = React.memo<
   // }, [editMode, replace]);
 
   const scheduledQueryGroupTableData = useMemo(() => {
-    const policyWithoutEmptyQueries = produce<
-      NewPackagePolicy,
-      OsqueryManagerPackagePolicy,
-      OsqueryManagerPackagePolicy
-    >(newPolicy, (draft) => {
-      draft.inputs[0].streams = filter(['compiled_stream.id', null], draft.inputs[0].streams);
-      return draft;
-    });
+    const policyWithoutEmptyQueries = produce<NewPackagePolicy, OsqueryManagerPackagePolicy>(
```

Comment:
Just update to the new type

Reply:
Weird that there were two `OsqueryManagerPackagePolicy` generics 🤔